drivers/d3d12/SCsub

```diff
@@ -43,8 +43,16 @@ if env["use_pix"]:
 
 # Mesa (SPIR-V to DXIL functionality).
 
-mesa_dir = (env["mesa_libs"] + "/godot-mesa").replace("\\", "/")
-mesa_gen_dir = (env["mesa_libs"] + "/godot-mesa/generated").replace("\\", "/")
+mesa_libs = env["mesa_libs"]
```

Comment:
It's a pre-existing issue, but I noticed that with relative paths, it breaks as it takes them relative to `drivers/d3d12/`:
```
FileNotFoundError: [Errno 2] No such file or directory: '/home/akien/Godot/godot/drivers/d3d12/bin/build_deps/mesa/godot-mesa/VERSION':
```